packages/consent/consent-wrapper-onetrust/src/lib/__tests__/onetrust-api.test.ts

```diff
@@ -0,0 +1,88 @@
+import './onetrust-globals.d.ts'
+
+import { getConsentedGroupIds, getGroupData } from '../onetrust-api'
+
+beforeEach(() => {
+  // @ts-ignore
+  delete window.OneTrustActiveGroups
+  // @ts-ignore
+  delete window.OneTrust
+})
+
+describe(getConsentedGroupIds, () => {
+  it('should return an empty array if no groups are consented', () => {
```

Comment:
Is this test named incorrectly?

Reply:
Hmm... it looks good to me eyes. What am I missing?

Reply:
I'm probably missing something, but the test says should return an empty array(...), and then we're expecting a non-empty array?